MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>Rather than manually make people populate this stuff consider having something to get a list of subscriptions and Sentinel workspaces they can just pick - see the AzureSentinel stuff I added to msticpy this week for examples.</p><p> </p><p>Also its easier if you just ask for the Resource ID and parse out these elements, that way someone can just copy and paste that out of the URI when they have the Sentinel workspace open in the Azure Portal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
updated with to use msticpy azure sentinel API